draft-gazdag-x509-hash-sigs.md

```diff
@@ -144,7 +167,33 @@ i.e. either 24, 32 or 64 bytes. The height of a single tree is typically given
 by the parameter 'h'. The number of levels of trees is either called 'L' (HSS)
 or 'd' (XMSS, XMSS^MT, SPHINCS+).
 
-# Subject Public Key Algorithms
+# Use Cases of HBS in X.509
+
+As many cryptographic algorithms that are considered to be quantum-resistant,
+HBS have several pros and cons regarding their practical usage. On the positive side
+they are considered to be secure against a classical as well as a quantum adversary,
+a secure instantiation of HBS may always be built as long as a cryptographically
+secure hash function exists. While key generation and signature generation may
+take longer than classical alternatives, fast and minimal verification routines
+can be built. The major negative aspect is the statefullness of several HBS.
+Private keys always have to be handled in a secure manner, but stateful HBS
+necessitate a special treatment of the private key in order to avoid
+security incidents like signature forgery [MCGREW], [NISTSP].
+
+As a secure environment MUST be used for key generation and key management,
+HBS are still a proper solution for root CA certificates. HBS MAY be used for
```

Comment:
Mein Vorschlag: 

... in order to avoid security incidents like signature forgery [MCGREW], [NISTSP]. Therefore, for stateful HBS, a secure environment MUST be used for key generation and key management.

Note that, in general, root CAs offer such a secure environment and the number of issued signatures (including signed certificates and CRLs) is often moderate due to the fact that many root CAs delegate OCSP services or the signing of end-entity certificates to other entities (such as subordinate CAs) that use non-stateful signature schemes for their signatures. Therefore, many root CAs should be able to handle the required state management, and stateful HBS are still a proper solution for usage there. 

HBS MAY also be used for subordinate CA certificates, but special and careful consideration MUST be taken into account for proper key management. HBS MUST NOT be used for end-entity certificates.

We include some further remarks on this issues in the section on security considerations.